src/app/cluster/cluster-details/component.ts

```diff
@@ -344,7 +351,7 @@ export class ClusterDetailsComponent implements OnInit, OnDestroy {
   }
 
   isOPAEnabled(): boolean {
-    return !!this.cluster.spec.opaIntegration && !!this.cluster.spec.opaIntegration.enabled;
+    return this.cluster.spec.opaIntegration.enabled;
```

Comment:
This actually can produce null pointer errors if the `opaIntegration` will be undefined.

Reply:
I think @floreks wanted to make it look like this: `!!this.cluster.spec.opaIntegration && this.cluster.spec.opaIntegration.enabled`.